### DISCUSSION TITLE
GraphQL for specific roles only

### DISCUSSION THREAD

**Discussion by RageBrahimi** [directus/directus#7213] [Feature Requests]
Hi,

I don’t know if this is possible, but thought I’d ask or just throw the idea out there.

Is it possible to disable the GraphQL endpoint for specific roles? Not sure if there’s currently a workaround for this.

**Comment by rijkvanzanten** [comment]
No, this isn't currently possible

**Reply by RageBrahimi** [reply]
Aha fair enough! Thanks for the reply

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
